lib/internal/histogram.js

```diff
@@ -65,26 +64,25 @@ class Histogram {
     if (percentile <= 0 || percentile > 100)
       throw new ERR_INVALID_ARG_VALUE.RangeError('percentile', percentile);
 
-    return this.#handle ? this.#handle.percentile(percentile) : undefined;
+    return this[kHandle] ? this[kHandle].percentile(percentile) : undefined;
   }
 
   get percentiles() {
     this.#map.clear();
-    if (this.#handle)
-      this.#handle.percentiles(this.#map);
+    if (this[kHandle])
+      this[kHandle].percentiles(this.#map);
     return this.#map;
   }
 
   reset() {
-    if (this.#handle)
-      this.#handle.reset();
+    if (this[kHandle])
+      this[kHandle].reset();
   }
 
   [kDestroy]() {
-    this.#handle = undefined;
+    this[kHandle] = undefined;
   }
 
-  get [kHandle]() { return this.#handle; }
```

Comment:
@aduh95 why did u suggest to remove this btw?

Reply:
The getter is useful only if you revert back to using private properties. When you define `this[kHandle]` in the constructor, it defines a "public" property, therefore a getter is not useful (and having one wouldn't work, hence the [CI error](https://github.com/nodejs/node/runs/1524430853) you got in 152f1fb). The consequence of using a public vs private property means consumers may assign a different value, which may or may not be a concern.

Reply:
thanks for explaining why that run failed